src/Plugin/Validators/EmptyCell.php

```diff
@@ -78,12 +104,136 @@ public function validateRow($row_values) {
     }
     else {
       $validator_status = [
-        'case' => 'No empty values found in required column(s)',
+        'case' => self::$mapping['case-valid']['dev-case'],
         'valid' => TRUE,
         'failedItems' => [],
       ];
     }
+
     return $validator_status;
   }
 
+  /**
+   * Processes failed validation from EmptyCell into a render array.
+   *
+   * @param array $failures
+   *   An associative array that stores the validation failures by the
+   *   EmptyCell validator. It is keyed by the line number of the input
+   *   file where validation failed, and the value is an associative array
+   *   returned by the validator. Here is the overall structure of $failures:
+   *   - [LINE NUMBER]:
+   *     - 'case': a developer-focused string describing the case checked.
+   *     - 'valid': FALSE to indicate that validation failed.
+   *     - 'failedItems': an array of items that failed:
+   *       - 'empty_indices': A list of column indices in the line which were
+   *         checked and found to be empty.
+   * @param array $tokens
+   *   [OPTIONAL] An array of values to use for token replacement.
+   *   @see EmptyCell::$mapping
+   *   The following token keys will substitute the entire existing case message
+   *   to the user with the value of that token.
+   *   - 'case-empty-value': the message when a specific row-column does not
+   *     contain a value.
+   * @param array $metadata
+   *   An array of additional metadata (or contextual information) needed by the
+   *   process method. Here, the following keys are expected:
+   *   - 'column_headers': This contains an array of headers defined in the
+   *     importer. The index in this array MUST match the position
+   *     (starting with 0) of the column in the input file.
+   *     Eg: 'column_headers' => [
+   *           '2' => 'Header 2', // Header of column #3
+   *           '4' => 'Header 4', // Header of column #5
+   *         ].
+   *
+   * @return array
+   *   A render array of type unordered list which is used to display feedback
+   *   to the user about the case(s) that failed and the failed items from the
+   *   input file. This unordered list will include a table that lists the row
+   *   and column combinations with empty cells. It has the following headers:
+   *   - 'Line Number'
+   *   - 'Column(s) with empty value'
```

Comment:
Just another suggestion to make this doc consistent with `GermplasmNameExists`:

```suggestion
   *   A render array of type "unordered list" used to display feedback to the
   *   user about the validation failure, where:
   *   - The 'title' is a sentence describing the case triggered
   *   - The 'items' include a table that lists the row and column combinations
   *     with empty cells. It has the following headers:
   *     - 'Line Number'
   *     - 'Column(s) with empty value'
```